uktrade/data-hub-frontend
# Add default placeholder to typeahead

## Description of change

This PR adds default placeholder text to the typeahead. This way, should any typeaheads not have custom placeholder messages, the users are still clear that they need to type in order to bring up suggestions. 

## Test instructions

On this branch, spin up storybook and go to the typeahead entry. Remove the placeholder provided. On refresh, you should see that the typeahead has the placeholder text of "Start typing" 


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
